packages/api/src/controllers/asset.test.ts

```diff
@@ -316,5 +324,58 @@ describe("controllers/asset", () => {
         },
       });
     });
+    describe("chunked upload", () => {
+      jest.setTimeout(10000);
```

Comment:
Comment this out before merging. Or if the tests are really taking that long, should tune them so they're faster.